SampleApps/ReactNativeTest/ios/EmptySwiftFile.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  EmptySwiftFile.swift
+//  ReactNativeTest
+//
+//  Created by Cannillo, Sammy on 5/18/21.
+//
+
+import Foundation
```

Comment:
Do we need both `Empty.swift` and `EmptySwiftFile.swift` or can we remove one of them?